breakout_1.md

```diff
@@ -0,0 +1,182 @@
+# Breakout Room #1
+
+## Background
+You and your collaborators have made a recent discovery that will surely result in winning the prestigious
+Nobel Peach Prize, awarded to the group of biomechanists with the best peach-related figure made entirely with 
+code! Your team has been working for months and have just a few bugs to fix before submitting the figure to 
+Nature for publication. Your team will be able to quickly fix these bugs in parallel thanks to Git, a version control 
+system.
+
+The code used to generate your figure is comprised of an upper-level script `ASB_Tutorial/main_script.m`, which 
+calls five numbered subscripts (e.g. `ASB_Tutorial/subscripts/script_1.m`). The data required to generate your 
+figure is stored as CSV files at `ASB_Tutorial/data/`. 
+
+If you need help remembering the git functions, refer to `ASB_Tutorial/git_cheatsheet.md`.
+
+## Objective
+Work together to fix the bug in each subscript and generate the figure by running `main_script.m`.
+
+## 1. Organize
+Your project leader has added you as collaborators on the remote repository on Github. If this hasn't happened yet, 
+request help from the Tutorial Team.
+
+#### a. Assign Tasks
```

Comment:
Why do you go down 2 levels here?

It makes later be the 4th level (4 ####), why not just 3?

Reply:
> There's nothing about deleting the branches. Do we want them to do that now?

I think so. Then they can make new branches for the next breakout room.